sdk/monitor/Azure.ResourceManager.Monitor/src/Generated/DataCollectionRuleData.cs

```diff
@@ -74,13 +78,7 @@ internal DataCollectionRuleData(ResourceIdentifier id, string name, ResourceType
         /// <summary> The resource ID of the data collection endpoint that this rule can be used with. </summary>
         public ResourceIdentifier DataCollectionEndpointId { get; set; }
         /// <summary> Metadata about the resource. </summary>
-        internal DataCollectionRuleMetadata Metadata { get; }
-        /// <summary> Azure offering managing this resource on-behalf-of customer. </summary>
-        public string MetadataProvisionedBy
-        {
-            get => Metadata?.ProvisionedBy;
-        }
```

Comment:
This removal is a breaking change. Please create a partial class of this class and put it in the `Customized` or `Custom` directory with and only with this removed property on that partial class?
We should already have a `Customized` or `Custom` directory, just put it there

Reply:
Also please add a `[EditorBrowsable(EditorBrowsableState.Never)]` attribute on the property to hide it in our SDK